.gitlab/ci/build_butte.yml

```diff
@@ -14,9 +14,10 @@
     - shell
     - butte
   stage: build
-  extends: .build_butte_script
-  except:
-    - /_bnone/
+  extends: .build_blueos_3_ppc64le_ib_script
+  only:
+    variables:
+      - $UMPIRE_CI_BUTTE == "ON"
```

Comment:
In butte case, we want the jobs to be deactivated by default. So the only condition applies.
It reads like:
Jobs using `.build_butte` property will be run `only` _if_ `$UMPIRE_CI_BUTTE == "ON"`.

Reply:
Note: major difference between only and except:
 - In `only:`, all categories (refs, variables) must have at least one condition fulfilled.
 - In `except:`, one fulfilled condition is enough to prevent the job from running.

https://docs.gitlab.com/ee/ci/yaml/README.html#onlyexcept-advanced

Reply:
Okay I understand. I was reading the statement as a variable assignment, not as a condition. Thanks for clarifying.